client/src/Movies/Movie.js

```diff
@@ -11,7 +11,8 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
perfect! it's not necessary, but to keep showing examples of what destructuring looks like for when you start seeing it more often, you could also destructure this value off of the params by declaring `const {id} = this.props.match.params` and `id` will store the same value